commit/report.go

```diff
@@ -38,7 +39,7 @@ func (p *Plugin) ShouldAcceptAttestedReport(
 ) (bool, error) {
 	decodedReport, err := p.reportCodec.Decode(ctx, r.Report)
 	if err != nil {
-		return false, fmt.Errorf("decode commit plugin report: %w", err)
+		return false, fmt.Errorf("decode merkleroot plugin report: %w", err)
```

Comment:
This is still in the root plugin right?

Reply:
yes, accidental replace all that happened when refactoring.